horizons/main.py

```diff
@@ -557,13 +557,10 @@ def edit_game_map(saved_game_name):
 	@return: bool, whether loading succeeded
 	"""
 	saved_games = SavegameManager.get_saves()
-	saved_game_path = _find_map(saved_game_name, saved_games)
-	if not saved_game_path:
+	map_name = _find_map(saved_game_name, saved_games)
+	if not map_name:
 		return False
 
-	accessor = SavegameAccessor(saved_game_path, False)
```

Comment:
> [...] possibly because as it is now, loading a map into the editor just loads the map's template.

Yes, I think that's the way this feature should work. And if we have a test that confirms this behaviour, we're done. Simple as that.